generate_R_table.py

```diff
@@ -68,8 +69,13 @@ def main():
     # answers to all the questions
     # Key = item number;
     # value = (correct_button_code, LeftorRight)
-    answer_key = dict_from_table(read_table(file_names['Question key filename']),
-                                                    paired=False)
+    
+    # THE TWO LINES BELOW ARE COMMENTED BECAUSE THEY DON'T WORK CORRECTLY.
+    # THE LINE BELOW THEM IS A QUICK FIX (ALSO SEE LINE 396). 
+    #answer_key = dict_from_table(read_table(file_names['Question key filename']),
+    #                                               paired=False)
```

Comment:
I don't know, actually. I am afraid of diving into the script's guts - I remember trying at it about a year ago, and I couldn't really get through (mostly because I rely on stepping through the code in the debugger to really understand how it works, and you use generators quite a lot, which makes it harder)